syntax = "proto3";

package Protocol;

message PlayerInfo{
    int32 playerId = 1;
}